pkg/kafka/auth/options.go

```diff
@@ -99,6 +106,17 @@ func addTLSFlags(configPrefix string, flagSet *flag.FlagSet) {
 		"Path to the TLS Key for the Kafka connection")
 }
 
+func addPlainTextFlag(configPrefix string, flagSet *flag.FlagSet) {
+	flagSet.String(
+		configPrefix+plainTextPrefix+suffixPlainTextUserName,
+		defaultPlainTextUserName,
+		"The plaintext Username for SASL/PLAIN authentication")
```

Comment:
shouldn't we call it basic like in other flags?

Reply:
Kafka and Sarama always mentioned it as Plaintext. Shall I change it to basic?
I can replace "plainText" to "basicAuth"

Reply:
I'd keep it as plaintext to match Kafka documentation.